typings/index.d.ts

```diff
@@ -16,5 +16,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/// <reference types="vite/client" />
-/// <reference types="vite-plugin-svgr/client" />
+declare module '*.svg' {
+  import * as React from 'react'
+
+  const ReactComponent: React.FunctionComponent<React.ComponentProps<'svg'> & { title?: string }>
```

Comment:
are you sure that SVG supports the title attribute?

Reply:
The title attribute is a global attribute so it can be used on all HTML elements (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes)

Having said that, it's not actually used in the implementation, so I will remove it for clarity